vendor/engines/saas/app/jobs/analytics/page_job.rb

```diff
@@ -0,0 +1,13 @@
+module Analytics
+  class PageJob < AnalyticsJob
+    action "page"
+
+    def payload(params)
+      {
+        user_id: params['current_user']['id'] || "Anonymous",
+        name: params['url'],
+        properties: { url: params['url'] }
+      }
```

Comment:
`user_id: params['current_user']['id'] || "Anonymous",`

I think this will throw if current_user is nil, I think you need to check both